outputs.tf

```
@@ -1,5 +1,5 @@
 output "saml_identity_provider_arn" {
   description = "ARN of the SAML Identity Provider"
-  value       = join("", aws_iam_saml_provider.this.*.arn)
+  value       = aws_iam_saml_provider.this.*.arn
```

It technically works, but since this resource is no longer a list, let's remove the `.*` syntax.
```suggestion
  value       = aws_iam_saml_provider.this.arn
```

---

Your slack message asked for the addition to .gitignore for test/go.*, but I was thinking of deletion and those lines had already been removed.  I've made the addition.

Corrected the 'value' so it no longer represents a list.  I didn't notice or forgot that change in change in other reviews.  Good to know.

---

Oh, yes. The missing piece is that when we run `make test` those files are generated automatically.